app/src/pages/inside/stepPage/modals/makeDecisionModal/makeDecisionModal.jsx

```diff
@@ -295,6 +297,7 @@ const MakeDecision = ({ data }) => {
             isBulkOperation={isBulkOperation}
             collapseTabsExceptCurr={collapseTabsExceptCurr}
             isNarrowView={!collapsedRightSection}
+            debugMode={debugMode}
```

Comment:
What the purpose to detect mode on top level component and passing it through the props? Seems it can be done directly in necessary component.